kraken/kubernetes/client.py

```diff
@@ -1,13 +1,19 @@
 from kubernetes import client, config
 from kubernetes.client.rest import ApiException
 import logging
+import kraken.invoke.command as runcommand
+import json
+
+kraken_node_name = ""
 
 
 # Load kubeconfig and initialize kubernetes python client
 def initialize_clients(kubeconfig_path):
     global cli
     config.load_kube_config(kubeconfig_path)
     cli = client.CoreV1Api()
+    global kraken_node_name
```

Comment:
I think this line and the following line can be removed as kraken_node_name is initialized previously (line 7 of client.py)

Reply:
Took those lines out